razorpay/utility/utility.py

```diff
@@ -14,12 +14,28 @@ def verify_payment_signature(self, parameters):
         order_id = str(parameters['razorpay_order_id'])
         payment_id = str(parameters['razorpay_payment_id'])
         razorpay_signature = str(parameters['razorpay_signature'])
-
+       
         msg = "{}|{}".format(order_id, payment_id)
-
+        
         secret = str(self.client.auth[1])
 
         return self.verify_signature(msg, razorpay_signature, secret)
+
+    def verify_payment_link_signature(self, parameters):
+        payment_id = str(parameters['razorpay_payment_id'])
+        payment_link_id = str(parameters['payment_link_id'])
+        payment_link_reference_id = str(parameters['payment_link_reference_id'])
+        payment_link_status = str(parameters['payment_link_status'])
+        razorpay_signature = str(parameters['razorpay_signature'])
+
+        msg = "{}|{}|{}|{}".format(payment_link_id, payment_link_reference_id, payment_link_status, payment_id)
+        
+        if 'secret' in parameters.keys() :
+          secret = str(parameters['secret'])
+        else:
+          secret = str(self.client.auth[1])
```

Comment:
This could be rewritten as

```
secret = str(parameters['secret']) if 'secret' in parameters.keys() else str(self.client.auth[1])
```

It's fine either ways.